src/command/toggleattributecommand.js

```diff
@@ -30,43 +31,29 @@ export default class ToggleAttributeCommand extends Command {
 		/**
 		 * Attribute that will be set by the command.
 		 *
+		 * @readonly
 		 * @member {String}
 		 */
 		this.attributeKey = attributeKey;
 
 		/**
```

Comment:
Hm... I'll be still working on the `ToggleAttributeCommand` so I'll see if I can put this differently.